src/screens/ForgotPassword/ForgotPassword.style.js

```diff
@@ -26,8 +20,6 @@ export default StyleSheet.create({
   textStyle: {
     fontSize: theme['text-primary-size'],
     fontWeight: theme['font-medium'],
-    color: theme['text-primary-color'],
```

Comment:
Please review this. I'm not sure if it is correct, but `text-primary-color` you've used in most cases can be replaced with the original `text-hint-color` or by passing `appearance='hint'` to Text props.